lib/elixir/lib/calendar.ex

```diff
@@ -1025,6 +1077,24 @@ defmodule NaiveDateTime do
     end
   end
 
+  @doc """
+  Compares two `NaiveDateTime` structs. Returns :gt if first is later than the second
+  and :lt for vice versa. If the two NaiveDateTime are equal :eq is returned
+
+  ## Examples
+
+    iex> ndt1 = NaiveDateTime.from_erl!({{2016, 4, 16}, {13, 30, 15}})
+    ~N[2016-04-16 13:30:15]
+    iex> ndt2 = NaiveDateTime.from_erl!({{2016, 4, 28}, {16, 19, 25}})
+    ~N[2016-04-28 16:19:25]
+    iex> NaiveDateTime.compare(ndt1, ndt2)
+    :lt
```

Comment:
These examples should be indented 4 spaces (they're indented 2) in order to act as doctests. This way they won't even get formatted as code.